### PR TITLE
feat: added prop to call http request with more parameters

### PR DESCRIPTION
**Pull request by wellingtonjr3873**
## Loom

deco-pr
deco-pr-2

### CODE REVIEW

File: vtex/loaders/intelligentSearch/productListingPage.ts [hunk 1]
```diff
@@ -143,12 +148,15 @@ export interface Props {
...
-
   /**
    * @hide true
    * @description The URL of the page, used to override URL from request
    */
   pageHref?: string;
+  aditionalFieldsInQuery?: {
+    label: string,
+    value: string
+  }[]
```
Comment (igorbrasileiro) [inline vtex/loaders/intelligentSearch/productListingPage.ts:159]:
```suggestion
/** @title {{{label}}} - {{{value}}} */
interface AditionalQueryFields {
    label: string,
    value: string
}

/// ...
  aditionalFieldsInQuery?: AditionalQueryFields[]
```

By doing the form in the admin UI have a better name/title that's readable for humans

Comment (igorbrasileiro) [reply vtex/loaders/intelligentSearch/productListingPage.ts:159]: Also, this prop was added to allow `showSponsored` and `placement`? Why not add it directly into the interface?

File: vtex/utils/intelligentSearch.ts [hunk 2]
```diff
@@ -45,6 +45,10 @@ interface Params {
   fuzzy: string;
   locale: string;
   hideUnavailableItems: boolean;
+  aditionalFieldsInQuery?: {
+    label: string,
+    value: string
+  }[]
```
Comment (igorbrasileiro) [inline vtex/utils/intelligentSearch.ts:51]: You can reuse the interface above

**Comment by guitavano** [comment]
UP @wellingtonjr3873

**Comment by guitavano** [comment]
Closed by inactivity.

**Pull request closed without merging**